src/NGen.cpp

```diff
@@ -91,41 +91,41 @@ int main(int argc, char *argv[]) {
                   << "Use \"all\" as explicit argument when no subset is needed." << std::endl;
 
         // Build and environment information
-        cout<<std::endl<<"Build Info:"<<std::endl;
-        cout<<"  NGen version: " // This is here mainly so that there will be *some* output if somehow no other options are enabled.
+        std::cout<<std::endl<<"Build Info:"<<std::endl;
+        std::cout<<"  NGen version: " // This is here mainly so that there will be *some* output if somehow no other options are enabled.
```

Comment:
It would be almost reasonable to put `using std::cout;` at the top of a `.cpp` file, rather than touching every print statement. If we're going to touch everything, then we may as well start using a function to provide the output stream.

Reply:
Well, yes, that kind of goes back to #574 and I concur... but... nevertheless I'm still puzzled how your previous set of commits compiles without this extra set of `std::`s.